.github/workflows/capture-repo-count.yml

```diff
@@ -0,0 +1,40 @@
+name: Capture Repo Count Metric
+
+on:
+  schedule:
+    - cron: "0 8 * * *" # Every day at 8am UTC
+  push:
+    branches: [snap-repo-metric]
+
+jobs:
+  capture:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Get Repo Count
+        uses: actions/github-script@v6
+        id: repo-count
+        with:
+          github-token: ${{ secrets.MACHINE_TOKEN }}
+          script: |
+            const writeFileAsync = require("util").promisify(require("fs").writeFile);
+            const repos = await github.paginate("GET /orgs/searchspring-implementations/repos");
```

Comment:
Looks like it maybe does 30 per page by default, maybe switch to 100 (via `per_page`) to reduce requests?
https://github.com/octokit/plugin-paginate-rest.js/#octokitpaginate

Reply:
Upped to 100